src/riak_dt.proto

```diff
@@ -107,6 +107,7 @@ message DtValue {
     optional sint64   counter_value = 1;
     repeated bytes    set_value     = 2;
     repeated MapEntry map_value     = 3;
+    repeated bytes    gset_value    = 4;
```

Comment:
Re-using `set_value` should be possible. I don't see the benefit of a separate field as `DataType` disambiguates it.